uwuno/src/test/java/com/learning/uwuno/unit/gameServiceTest.java

```diff
@@ -52,7 +52,7 @@ public void testGameService_addRoom() throws Exception {
     }
 
     @Test
-    public void testGameService_addPlayer() throws Exception {
+    public void testGameService_addPlayer() {
         room room = gameService.addRoom(testRoomName, false);
         assertThrows(NoSuchElementException.class, () -> {
```

Comment:
Is it possible you did this on a not too recent branch, I'm pretty sure all the service functions were changed to throw our own excetpions.

Reply:
It was merged from the old master